config/settings.yml

```diff
@@ -36,3 +36,8 @@ dor_services:
 sunet:
   id: ~ # NOTE: *without* @stanford.edu!
   password: ~
+
+# the URL and username/password used to login to goobi (stored in vault)
```

Comment:
Turns out they were not put into vault. See https://stanfordlib.slack.com/archives/C0RK5EM9N/p1658427869896549